src/plugins/embeddable/public/lib/test_samples/embeddables/contact_card/contact_card_embeddable_factory.tsx

```diff
@@ -56,6 +56,7 @@ export class ContactCardEmbeddableFactory
           <ContactCardInitializer
             onCancel={() => {
               modalSession.close();
+              // @ts-expect-error
```

Comment:
`undefined` is not assignable to `Partial<ContactCardEmbeddableInput>` @elastic/kibana-app-arch

Reply:
Maybe

```ts
reject(new Error('Cancelled!'));
```

Reply:
it's not the same as `resolve(undefined)`